packages/ui/src/pages/DashboardPage/components/AddSession.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { defineEmits } from "vue";
+
+const emit = defineEmits(["add-session"]);
+
+const addSession = () => {
+  emit("add-session");
+};
+</script>
+
+<template>
+  <div
+    class="flex flex-col w-72 m-2 shrink-0 p-6 border rounded-xl border-black justify-center items-center bg-gray-200"
+  >
+    <FontAwesomeIcon
+      :icon="['fas', 'plus']"
+      class="handle cursor-pointer fa-8x"
```

Comment:
I don't think this needs `class="handle"`, I assume this was copied from the session card icon which is used to drag the sessions?